README.md

```diff
@@ -45,7 +321,7 @@ When you reach point where you have `exampleName.cer` file, import it to Keychai
 File -> Import Items... -> Chose `exampleName.cer` 
 
 Next, export the certificate you just imported as `exampleName.p12`.
-Note: you can just go with empty password by pressing Enter. If you enter a password rememeber it.
+Note: you can just go with empty password by pressing Enter. If you enter a password remember it.
```

Comment:
Note: you can just go with no password by pressing Enter. If you enter a password, remember it.